core/src/main/scala/org/apache/spark/ui/SparkUI.scala

```diff
@@ -60,6 +60,8 @@ private[spark] class SparkUI private (
 
   var appId: String = _
 
+  var appSparkVersion = ""
```

Comment:
we are no longer setting this for the liveUI so it either needs to be default here to org.apache.spark.SPARK_VERSION or we need to call the setAppSparkVersion in SparkContext.